rules/CodeQualityStrict/Rector/Variable/MoveVariableDeclarationNearReferenceRector.php

```diff
@@ -297,6 +298,15 @@ private function hasCall(Node $node): bool
                 return false;
             }
 
+            if (! function_exists($funcName)) {
+                return true;
+            }
+
+            $r = new ReflectionFunction($funcName);
+            if (! $r->isInternal()) {
+                return true;
+            }
```

Comment:
Here the `ReflectionProvider` should be used instead, so we have static reflection on board here :+1:

Reply:
implemented.